css/main.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700&family=Roboto:wght@100&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  font-family: 'Inter', sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.headline-sec .profession {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.job {
+  margin-top: 3rem;
+}
+
+.title {
+  text-align: center;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.5rem;
+}
+
+.nav-toggle {
+  background: none;
+  border: none;
+  cursor: pointer;
+  right: 1em;
+  top: 1em;
+}
+
+.header .welcome-logo {
+  font-size: 1.25rem;
+  font-weight: 600;
+}
+
+.name {
+  margin-top: 10px;
+  color: #172b4d;
+}
+
+.job-title {
+  color: #36b37e;
+}
+
+.text {
+  font-weight: 400;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  margin: 1.8rem 2rem 0 1.2rem;
+  text-align: center;
+  color: #172b4d;
+}
+
+.social-contact {
+  margin-top: -90px;
+}
+
+.social-contact .social {
+  display: flex;
+  flex-direction: row;
+  margin-top: 10rem;
+  justify-content: center;
+  text-align: center;
+}
+
+@media (max-width: 768px) {
+  .welcome-logo {
+    margin-left: 3px;
+  }
+
+  .nav {
+    margin-right: 15px;
+    cursor: pointer;
+  }
+
+  .header {
+    margin: 20px;
+  }
+
+  .headline-sec {
+    display: flex;
+    flex-direction: column;
+    background-image: url(/img/bg-header-illsutration-img/header-illsutration-mobile.svg);
```

Comment:
- [ ] Kindly make sure the background image is properly positioned to the right. ([See the Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468))